common/crypto/platform/openssl/crypto.cpp

```diff
@@ -14,6 +14,14 @@
 
 #include <common/crypto.hpp>
 
+#include <common/crypto/platform/openssl/config.h>
```

Comment:
Probably an error while rebasing. This should be:

```
#include <common/crypto/platform/openssl/openssl_config.h>
```

Which is now few lines below :point_down:

Reply:
Oh, yes! thx!